docs/dashboards/entity-dashboard.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[detection-entity-dashboard]]
+= Entity dashboard
+
+The Entity Analytics dashboard provides a centralized view into emerging insider threats - including host risk, user risk, and notable anomalies from within your network. Use these analytics to triage emerging threats, investigate, and respond.
+
+
+.Requirements
+[sidebar]
+--
+The following is required to display the Entity Analytics dashboard: 
+
+* A https://www.elastic.co/pricing/[Platinum subscription] or higher.
+* To display host and user risk scores, the host risk score and user risk score features must be enabled. You can do this directly from the dashboard. For more information, refer to Enable host risk score and <<deploy-user-risk-score, Enable user risk score>>.
```

Comment:
I'm sure you know, but just flagging that Enable host risk score is not a link yet.

Reply:
Also, the link that does exist (pointing to [here](https://security-docs_2565.docs-preview.app.elstc.co/guide/en/security/master/user-risk-score.html#deploy-user-risk-score)) doesn't seem that helpful because the target page is explaining the _old long way_ of enabling the feature.

This note says "you can do this directly from the dashboard" but doesn't say exactly how to do that, so a reader might think, OK, let me follow this link for more info on how to do that. But all they need to do is just push the button right on the dashboard, which we never tell them to do.